api_app/tests_ma/test_services/test_cost_service.py

```diff
@@ -134,13 +95,48 @@ def test_query_tre_costs_with_granularity_daily_returns_correct_cost_report(clie
     assert cost_report.workspaces[0].costs[2].date == date(2022, 5, 3)
     assert cost_report.workspaces[1].id == "d680d6b7-d1d9-411c-9101-0793da980c81"
     assert cost_report.workspaces[1].name == "the workspace display name2"
-    assert len(cost_report.workspaces[1].costs) == 3
+    assert len(cost_report.workspaces[1].costs) == 4
     assert cost_report.workspaces[1].costs[0].cost == 4.8
     assert cost_report.workspaces[1].costs[0].date == date(2022, 5, 1)
     assert cost_report.workspaces[1].costs[1].cost == 5.8
     assert cost_report.workspaces[1].costs[1].date == date(2022, 5, 2)
     assert cost_report.workspaces[1].costs[2].cost == 6.8
     assert cost_report.workspaces[1].costs[2].date == date(2022, 5, 3)
+    assert cost_report.workspaces[1].costs[2].currency == "USD"
+    assert cost_report.workspaces[1].costs[3].cost == 16.8
+    assert cost_report.workspaces[1].costs[3].date == date(2022, 5, 3)
+    assert cost_report.workspaces[1].costs[3].currency == "ILS"
+
+
+def __get_daily_cost_management_query_result():
```

Comment:
nit: I would try to split into the rows into specific scenarios and test cases vs using a big list of rows for all scenarios - easier to maintain